package.json

```diff
@@ -115,6 +118,9 @@
     "webpack-cli": "4.2.0",
     "webpack-dev-server": "4.5.0"
   },
+  "engines": {
+    "node": ">=16 < 18"
```

Comment:
The version of electron etcher currently uses (12.2.3) runs nodejs 14.16.0, so it probably makes sense for the install/compile/etc to at least match 14.x

Reply:
@Page- turns out there is a bug in npm < 8 where it changes git URLs to git:// vs. https://. Since there's now SSH keys injected, private repos fail.